practice-two/src/App.jsx

```diff
@@ -25,6 +25,24 @@ const App = () => {
   // State for storing the favorite product IDs
   const [favoriteProductIDs, setFavoriteProductIDs] = useState([]);
 
+  // State for storing the selected product's ID for details view
+  const [selectedProductID, setSelectedProductID] = useState(null);
+
+  // Function to handle selecting a product card
```

Comment:
Check the comment format
it's should be 
```
/**
 * function do sth
 * @params ....
 * @return ....
 */
```

right? Did I told you before

Reply:
yes you said, updated the functions